configs/PicoFightingBoard/BoardConfig.h

```diff
@@ -0,0 +1,202 @@
+// It is recommended to disable the `TURBO_LED_PIN` by setting it to `-1` if you are sensitive of flashing lights.
+// The default `BOARD_LEDS_PIN` pin is set to `28`.
+// The board LED pin will allow you to connect addressible RGB LEDs on the Pico.
+// Addressible RGB LEDs should be connected to the `VBUS` pin (#40), an avalible ground pin and the defined `BOARD_LEDS_PIN`.
+// Special note - You should only ever use addressible RGB LEDs that are rated for 5v operation on the Pico.
+// The defualt `LED_BRIGHTNESS_MAXIMUM` value is `50`.
+// This will change how bright the LEDs are with `0` being off and `100` being full brightness.
+// The minimum `LED_BRIGHTNESS_MAXIMUM` value is `0`.
+// The maximum `LED_BRIGHTNESS_MAXIMUM` value is `100`.
+// The defualt `LED_BRIGHTNESS_STEPS` value is `5`.
+// This will change how many brightness steps there are when increasing or decreasing the brightness of the LEDs via the onboard shortcut.
+// It is recommend to keep this value at `5` or `10`.
+// The defualt `LED_FORMAT` is `LED_FORMAT_GRB`.
+// You can also choose the `LED_FORMAT` of `LED_FORMAT_RGB` if your addressible RGB LEDs are not working as intended.
+// The defualt `LEDS_PER_PIXEL` is set to `1`.
+// This will change how many addressible RGB LEDs there are per button.
+// The default LEDS_[BUTTON] is an order and has nothing to do with what GPIO pin something is connected to.
+// Unless you are planning on running custom animations I would recommmend you leave this as is.
+
+#define TURBO_LED_PIN -1
+
+#define BOARD_LEDS_PIN 15
+
+#define LED_BRIGHTNESS_MAXIMUM 200
+#define LED_BRIGHTNESS_STEPS 5
+#define LED_FORMAT LED_FORMAT_GRB
+#define LEDS_PER_PIXEL 2
+
+#define LEDS_DPAD_LEFT   11
+#define LEDS_DPAD_DOWN   10
+#define LEDS_DPAD_RIGHT  9
+#define LEDS_DPAD_UP     0
+#define LEDS_BUTTON_B3   8
+#define LEDS_BUTTON_B4   7
+#define LEDS_BUTTON_R1   6
+#define LEDS_BUTTON_L1   5
+#define LEDS_BUTTON_B1   1
+#define LEDS_BUTTON_B2   2
+#define LEDS_BUTTON_R2   3
+#define LEDS_BUTTON_L2   4
+#define LEDS_BUTTON_S1   -1
+#define LEDS_BUTTON_S2   -1
+#define LEDS_BUTTON_L3   -1
+#define LEDS_BUTTON_R3   -1
+#define LEDS_BUTTON_A1   -1
+#define LEDS_BUTTON_A2   -1
+
+
+// This is the Player LED section.
+// In this section you can specify if Player LEDs will be active, and, if active, which pins will be used for them.
+// The defualt is `PLED_TYPE_NONE` which will turn the Player LEDs off.
+// The default pin for each Player LED is `-1` which disables it.
+// To enable a `PLED#_PIN`, replace the `-1` with the GPIO pin number that is desired.
+// There are three options for `PLED_TYPE` currently:
+// 1 - `PLED_TYPE_NONE` - This will disable the Player LEDs
+// 2 - `PLED_TYPE_PWM` - This will enable the Player LEDs ( it is recommended to run through 3V3(OUT) with a resistor)
+// 3 - `PLED_TYPE_RGB` - This will enable the Player LEDs as addressible RGB LEDs (please not that this has not been implemented yet)
+
+#define PLED_TYPE PLED_TYPE_NONE
```

Comment:
This should be updated to:

PLED_TYPE PLED_TYPE_PWM to enable Player LED out by default